package.json

```
@@ -53,9 +53,10 @@
     "@storybook/addon-links": "7.6.17",
     "@storybook/react": "7.6.17",
     "@storybook/react-vite": "7.6.17",
-    "@testing-library/jest-dom": "6.4.2",
-    "@testing-library/react": "14.2.1",
-    "@testing-library/user-event": "13.5.0",
+    "@testing-library/dom": "^10.4.0",
```

```suggestion
    "@testing-library/dom": "10.4.0",
```
Mind keeping us at this version?